zipline/pipeline/engine.py

```diff
@@ -452,17 +461,17 @@ def _inputs_for_term(term, workspace, graph):
                 out.append(input_data)
         return out
 
-    def get_loader(self, term):
-        return self._get_loader(term)
-
-    def compute_chunk(self, graph, dates, assets, initial_workspace):
+    def compute_chunk(self, graph, dates, sids, initial_workspace):
         """
         Compute the Pipeline terms in the graph for the requested start and end
         dates.
 
+        This is where we do the actual work of running a pipeline.
+
         Parameters
         ----------
-        graph : zipline.pipeline.graph.TermGraph
+        graph : zipline.pipeline.graph.ExecutionPlan
```

Comment:
the variable name `graph` seems to get used for both of these types of objects.

Reply:
`ExecutionPlan` is a subclass of `TermGraph`, so `graph` is accurate for both of them.